sw/airborne/firmwares/rotorcraft/main_ap.c

```diff
@@ -305,60 +305,58 @@ void failsafe_check(void)
 {
 #if !USE_GENERATED_AUTOPILOT
   if (radio_control.status == RC_REALLY_LOST &&
-      autopilot_mode != AP_MODE_KILL &&
-      autopilot_mode != AP_MODE_HOME &&
-      autopilot_mode != AP_MODE_FAILSAFE &&
-      autopilot_mode != AP_MODE_NAV &&
-      autopilot_mode != AP_MODE_MODULE &&
-      autopilot_mode != AP_MODE_FLIP &&
-      autopilot_mode != AP_MODE_GUIDED) {
+      autopilot_get_mode() != AP_MODE_KILL &&
```

Comment:
@OpenUAS it will have exactly the same behavior than before, it is just a simple getter function that doesn't change the variable. The only possible inconsistency is if you start changing the mode without proper protection in multi-threaded environment or in interrupts. It is not the case at the moment (and hopefully it will never be the case!).